front/components/assistant/conversation/actions/AgentMessageActions.tsx

```diff
@@ -31,8 +33,8 @@ export function AgentMessageActions({
         setChipLabel("Thinking");
         break;
       case "acting":
-        if (agentMessage.actions.length > 0) {
-          setChipLabel(renderActionName(agentMessage.actions));
+        if (agentMessage.lightActions && agentMessage.lightActions.length > 0) {
```

Comment:
Type distinction should be enough, no? Not sure we need a different attribute name.
```suggestion
        if (agentMessage.actions && agentMessage.actions.length > 0) {
```